src/Plugin.php

```diff
@@ -112,6 +112,7 @@ public static function init() {
     WooCommerceSalutation::init();
     WooCommerceCheckout::init();
     PlusProducts::init();
+    Products::init();
```

Comment:
Since this PHP class is doing something very specific, I think this is a too general class name, so something similar to the existing `ProductAttributePageTitle`, maybe like `ProductPermalink`, might be a good idea.

Reply:
I took inspiration from other classes here such as `PlusProducts` which also sounds a bit wide... I'd adopt your suggestion in case you agree to create a special namespace for the whole Product domain classes, otherwise I created this one to start shaping up all possible future product related features from one place